config-controller/internal/controller/policy_controller.go

```diff
@@ -67,13 +67,16 @@ func (r *SecurityPolicyReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		return ctrl.Result{}, errors.Wrapf(err, "Invalid policy resource: namespace=%s, name=%s", req.Namespace, req.Name)
 	}
 
-	if err := r.PolicyClient.EnsureFresh(ctx); err != nil {
+	if err := r.CentralClient.EnsureFresh(ctx); err != nil {
 		return ctrl.Result{}, errors.Wrap(err, "Failed to refresh")
 	}
 
-	desiredState := policyCR.Spec.ToProtobuf()
+	desiredState, err := policyCR.Spec.ToProtobuf(r.CentralClient.GetNotifiers())
+	if err != nil {
+		return ctrl.Result{}, errors.Wrap(err, "Failed to convert policy to protobuf")
```

Comment:
This now needs to set the status in order to surface invalid notifier name errors to the user via the status field.